tests/table/hovered_table_store.test.ts

```diff
@@ -0,0 +1,86 @@
+import { Model, UID } from "../../src";
+import { HoveredTableStore } from "../../src/components/tables/hovered_table_store";
+import { TABLE_HOVER_BACKGROUND_COLOR } from "../../src/constants";
+import { createTable, setCellContent } from "../test_helpers/commands_helpers";
+import { makeStore } from "../test_helpers/stores";
+
+describe("Hovered Table Store", () => {
+  let hovered_table_store: HoveredTableStore;
+  let model: Model;
+  let sheetId: UID;
+
+  beforeEach(() => {
+    ({ model, store: hovered_table_store } = makeStore(HoveredTableStore));
+    hovered_table_store.clear();
+    sheetId = model.getters.getActiveSheetId();
+    createTable(model, "A1:B2");
```

Comment:
as a general rule, I don't like `beforeEach` with business stuff (updating the spreadsheet):

- you can't see and understand the full picture when looking at a test. You have to scroll up to see there's a table on A1:B2
- one day, someone will want to add a test but with another table config. So that person will add the table in the test, but then there are two tables while only one would be enough. The test won't be a minimal test

It's ok (better) to have the setup in each test, even if it's copy-pasted. I'd move `createTable(model, "A1:B2");` in each test